components/automate-ui/src/app/modules/infra-proxy/empty-state/empty-state.component.ts

```diff
@@ -0,0 +1,19 @@
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
+
+@Component({
+  selector: 'app-empty-state',
+  templateUrl: './empty-state.component.html',
+  styleUrls: ['./empty-state.component.scss']
+})
+
+
+
+export class EmptyStateComponent {
+    @Input() moduleTitle: string;
+    @Output() resetKeyRedirection = new EventEmitter<boolean>();
+
+    ResetKeyTabRedirection() {
+        this.resetKeyRedirection.emit(true);
+    }
```

Comment:
Consistent 2-space tabbing, camelCase, and there's some extra newlines above and below the `@Component` decorator.